packages/concrete5_theme/blocks/tutorial_related_list/templates/minimalist-list-view.php

```diff
@@ -0,0 +1,50 @@
+<?php
+defined('C5_EXECUTE') or die("Access Denied.");
+use Concrete\Package\Concrete5Docs\Page\PageInspector;
+use Concrete\Core\Support\Facade\Application;
+use Concrete\Core\Utility\Service\Text;
+use Concrete\Core\Page\Page;
+
+$app = Application::getFacadeApplication();
+
+if ($results) {
+
+    foreach ($results as $result) {
+        /** @var Page $result */
+        $username = t('Unknown');
+        $ui = UserInfo::getByID($result->getCollectionUserID());
+        if (is_object($ui)) {
+            $username = $ui->getUserDisplayName();
+        }
+        $inspector = new PageInspector($result);
+
+        $collectionDate = $result->getCollectionDatePublic();
+        if ($collectionDate) {
+            $date = strtotime($collectionDate);
+            $publishDate = date('m/d/Y', $date);
+        }
+
+        /** @var Text $th */
+        $th = $app->make('helper/text');
+        ?>
+
+        <div class="ccm-margin-bottom-20">
+            <a href="<?=$result->getCollectionLink()?>" style="margin-right: 10px; font-weight: bold; text-decoration: none;">
```

Comment:
Let's add a class for this instead of the inline styles, since it looks like you're already working on the concrete5 theme in some way.

Reply:
totally missed this one ooops!